apps/reaction/tests/util/makeRemoteExecutableSchema.js

```diff
@@ -0,0 +1,29 @@
+import { print } from "graphql";
```

Comment:
So @aldeed probably knows this stuff better than I do, but one of the reasons we didn't do Executable Schemas in the past was because it is incompatible with Federated Gateways (which we may or may not do in the future). Is this no longer so?

Reply:
I don't know if it's true, so I think this needs testing. If there isn't already an integration test for federation, it might be a good time to add one. Whether or not federation is part of the standard offerings, it should be fairly easy to continue supporting it for those who want to opt in.